src/components/CourseCard/EmailSettingsModal.jsx

```diff
@@ -118,6 +119,7 @@ class EmailSettingsModal extends Component {
                   {isSubmitting && '...'}
                 </>
               ),
+              className: classNames('save-email-settings-btn', { 'is-form-changed': isFormChanged }),
```

Comment:
out of curiosity, why is this code in blue :P

Reply:
Good question haha, I was wondering the same thing; thought there was a syntax error somewhere! 😄